ctapipe/utils/tests/test_json_2_fits.py

```diff
@@ -123,7 +123,10 @@ def test_traitlets_config_to_fits():
     app = MyApp()
     app.initialize()
     app.start()
-    assert(app.traitlets_config_to_fits() == True)
+    try:
+        app.traitlets_config_to_fits()
+    except:
+        assert()
```

Comment:
`assert()` means the same as `assert tuple()` Assert is not a function. You test if the empty tuple evaluates to True.